component/discovery/relabel/relabel.go

```diff
@@ -28,7 +28,7 @@ type Arguments struct {
 	// Targets contains the input 'targets' passed by a service discovery component.
 	Targets []discovery.Target `river:"targets,attr"`
 
-	// The relabelling rules to apply to the each target's label set.
+	// The relabelling rules to apply to each target's label set.
```

Comment:
@tpaschalis I am not 100% certain. Did you mean `the target's label set` or `each target's label set`?